internal/component/prometheus/exporter/cloudwatch/config.go

```diff
@@ -77,8 +81,14 @@ type StaticJob struct {
 	Namespace  string         `alloy:"namespace,attr"`
 	Dimensions Dimensions     `alloy:"dimensions,attr"`
 	Metrics    []Metric       `alloy:"metric,block"`
+	Period     time.Duration  `alloy:"period,attr,optional"`
+	Length     time.Duration  `alloy:"length,attr,optional"`
 	Delay      time.Duration  `alloy:"delay,attr,optional"`
-	//TODO: Remove NilToZero, because it is deprecated upstream.
+	// NOTE: This field is actually not supported as a configuration option in YACE!
+	// https://github.com/prometheus-community/yet-another-cloudwatch-exporter/blob/0c9677d91836f0a4150a55172a0ce5081574b407/docs/configuration.md?plain=1#L177
+	// It should either be removed from Alloy in some major release, or
+	// contributed to YACE.
+	// We currently patch it in toStaticJob func to make it work and not break existing configs.
```

Comment:
I think the docs might be out of sync with the code. The staticjob has the metrics struct, https://github.com/prometheus-common/yet-another-cloudwatch-exporter/blob/70fcc27577c09e582df1070674bda5e99d16cea6/pkg/config/config.go#L71-L79, which has NilToZero https://github.com/prometheus-common/yet-another-cloudwatch-exporter/blob/70fcc27577c09e582df1070674bda5e99d16cea6/pkg/config/config.go#L94-L102